x-pack/platform/plugins/shared/fleet/server/routes/cloud_connector/handlers.ts

```diff
@@ -38,7 +40,11 @@ export const createCloudConnectorHandler: FleetRequestHandler<
 
   try {
     logger.info('Creating cloud connector');
-    const cloudConnector = await cloudConnectorService.create(internalSoClient, request.body);
+    const cloudConnector = await cloudConnectorService.create(
+      internalSoClient,
+      // Type assertion is safe: schema validation ensures structure, service validates vars against CloudConnectorVars
+      request.body as unknown as CreateCloudConnectorRequest
```

Comment:
Can we type `request.body: TypeOf<typeof CreateAgentlessPolicyRequest.body>`? I've seen this pattern in pattern.